Add brief summary of possible ways to use ESMValTool (#194)
ESMValGroup/ESMValTool_Tutorial: _episodes/02-installation.md

We will first install Conda, and then ESMValTool. We end this chapter by testing
that the installation was successful.

> 
> ## ESMValTool Installation/Usage  Options - A quick summary

Reviewer: ```suggestion
```

>  Before we begin, here are all the possible ways in which you can use ESMValTool
>  depending on your level of expertise or involvement with ESMvalTool and 
>  associated software such as GitHub and Conda.

Reviewer: ```suggestion
Before we begin, here are all the possible ways in which you can use ESMValTool
depending on your level of expertise or involvement with ESMvalTool and 
associated software such as GitHub and Conda.
```

>  1) If you have access to a server where ESMValTool is already installed 
>  as a module, for e.g., the [CEDA JASMIN](https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool) 

Reviewer: ```suggestion
1. If you have access to a server where ESMValTool is already installed
as a module, for e.g., the [CEDA JASMIN](https://help.jasmin.ac.uk/article/4955-community-software-esmvaltool)
```

server, you can simply load the module with the following:

>  ```bash
>  module load esmvaltool
>  ```
> 
> After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
>  

Reviewer: ```suggestion
~~~bash
module load esmvaltool
~~~
After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
```

> 2) If you would like to install ESMValTool as a conda package, then this lesson 
>  will tell you how!
>  
>  3) If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
>  source installation in the lesson [Development and
>  contribution]({{ page.root }}{% link _episodes/08-development-setup.md %}) and 
>  in the [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html).
> 

Reviewer: ```suggestion
2. If you would like to install ESMValTool as a conda package, then this lesson
will tell you how!
3. If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
source installation in the lesson [Development and
contribution]({{ page.root }}{% link _episodes/08-development-setup.md %}) and
in the [documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html).
```

We will first install Conda, and then ESMValTool. We end this chapter by testing
that the installation was successful.

> ## ESMValTool Installation/Usage  Options - A quick summary
> Before we begin, here are all the possible ways in which you can use ESMValTool
> depending on your level of expertise or involvement with ESMvalTool and 
> associated software such as GitHub and Conda.
>
> 1) If you have access to a server where ESMValTool is already installed 
> as a module, for e.g., the ``CEDA JASMIN`` server, you can simply

> > load the module and start using ESMValTool with the following :

Reviewer: ```suggestion
> load the module with the following:
```

> ```bash
> module load esmvaltool

> > ```

Reviewer: ```suggestion
> ```

After loading ``esmvaltool``, we can start using ESMValTool. Please see the [next lesson]({{ page.root }}{% link _episodes/03-configuration.md %}). 
```

> 2) If you would like to install ESMValTool as a conda package, then this lesson 
> will tell you how!
> 

> > 3) If you are already familiar with ESMValTool and would like to start experimenting
> > with existing diagnostics, you can dowload the source code from the ESMvalTool Github
> > page, compile and run as given here (link missing).
> >

Reviewer: ```suggestion
```

> > 4) If you would like to start contributing to ESMvalTool, please see the instructions for 

Reviewer: ```suggestion
> 3) If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
```

> > 4) If you would like to start contributing to ESMvalTool, please see the instructions for 
> > source installation in the lesson [Development and

Reviewer: ```suggestion
> 3) If you would like to start experimenting with existing diagnostics or contributing to ESMvalTool, please see the instructions for 
> source installation in the lesson [Development and
```
```suggestion
> 4) If you would like to start contributing to ESMvalTool, please see the instructions for 
> source installation in the lesson [Development and
```
